### ISSUE TITLE
Question on the Poisson solvers' behaviour

### ISSUE THREAD

**Issue by smontefiori** [SmileiPIC/Smilei#408]
Dear developers,
hi! I was following the tutorial page (https://smileipic.github.io/tutorials/advanced_wakefield_electron_beam.html) concerning the field initialization for a relativistic electron bunch, in which a test namelist simulating a relativistic electron bunch entering a plasma in a 2D geometry is provided.

At some point in the page, the section "A subtlety: why ions are not present?" (see photo below) is present:
![immagine](https://user-images.githubusercontent.com/83040284/120109873-962b1780-c16b-11eb-8cfc-7b35ce7f76ff.png)

I have to admit that the main idea behind this consideration is quite smart, however I am confused. I mean, shouldn't the background behave as charged (in such a way to ensure a zero divergence electric field) only in case the poisson solvers were turned off? However, in the attached namelist, the relativistic one is still active and, thus, the plasma at rest is even more bound to explode, isn't it?

What am I missing?

I apologize in advance for the bother.
Thank you for your time and kindness.

**Comment by Z10Frank** [comment]
Hello, 
the relativistic Poisson solver acts only on the relativistic `Species` for which it is activated (there is a dedicated flag inside the `Species` block in the input namelist). 

Thus, for the solver of Eq. 62 (https://smileipic.github.io/Smilei/relativistic_fields_initialization.html) you will have as source term only the charge density of the relativistic Species, the electron bunch. This way, it will find the electromagnetic field only of the relativistic electron bunch.

For the immobile background electrons however, no solver is used, so zero initial electric field, which is equivalent to have a neutralizing superposed `Species`, as explained in the tutorial.

To summarize, by linear superposition at t=0 you will have a relativistic `Species` with its consistent field and an immobile `Species` that behaves as if there is an immobile neutralizing background.

Is it more clear now?

**Comment by smontefiori** [comment]
Uh, understood! Yes, you made it very clear!

Thank you so much, the issue is closed.
